css/device-catalog.css

```diff
@@ -0,0 +1,884 @@
+    padding-top: 23px;
+    padding-bottom: 23px;
+}
+.page-back:hover,
+.page-forward:hover {
+    background-color: #d9d9d9;
+}
+.page-back:active,
+.page-forward:active {
+    color: rgba(0, 0, 0, 0.3);
+}
+
+/* SIDE BAR */
+
+.side-bar {
+    max-width: 467px;
+    padding-left: 200px;
+    padding-right: 68px;
+
+    background-color: #efefef;
+}
+.side-bar h2 {
+    margin-top: 23px;
+    margin-bottom: 23px;
+}
+.side-bar legend {
+    font-family: 'Gilroyextrabold', 'Opensans', sans-serif;
+    font-size: 18px;
+    line-height: 24px;
+}
+.side-bar label {
+    position: relative;
+    padding-left: 43px;
+
+    font-family: 'Opensanslight', 'sans-serif';
+    font-size: 14px;
+    line-height: 40px;
+}
+.filter-price {
+    max-width: 201px;
+    border-style: none;
+    padding: 0;
+    margin-bottom: 40px;
+    margin-top: 80px;
+}
+.filter-color {
+    max-width: 201px;
+    border-style: none;
+    padding: 0;
+    margin-bottom: 40px;
+}
+.filter-bluetooth {
+    max-width: 201px; 
+    border-style: none;
+    padding: 0;
+    margin-bottom: 40px;
+}
+.filter-button {
+    font-family: 'Gilroyextrabold', 'Opensans', sans-serif;
+    font-size: 18px;
+    line-height: 24px;
+
+    text-transform: uppercase;
+    letter-spacing: 3px;
+}
+.container-color input[type="checkbox"] {
+    opacity: 0;
+    position: absolute;
+}
+.container-color label::before {
+    content: "";
+    display: inline-block;
+    position: absolute;
+
+    width: 23px;
+    height: 23px;
+
+    border: 2px solid;
+
+    left: 0;
+    top: -2px; 
+}
+.container-color label::after {
+    content: "";
+    display: inline-block;
+    position: absolute;
+
+    width: 21px;
+    height: 12px;
+    border-left: 4px solid;
+    border-bottom: 4px solid;
+
+    transform: rotate(-45deg);
+    left: 4px;
+    top: -2px;
+}
+.container-color input[type="checkbox"] + label::after {
+    content: none;
+}
+.container-color input[type="checkbox"]:checked + label::after {
+    content: "";
+}
+.container-color input[type="checkbox"]:hover + label::before,
+.container-color input[type="checkbox"]:hover + label::after {
+    opacity: 0.6;
+}
+.container-color input[type="checkbox"]:focus + label::before,
+.container-color input[type="checkbox"]:focus + label::after {
+    opacity: 1;
+}
+
+.container-bluetooth input[type="radio"] {
+    position: absolute;
+    opacity: 0;
+}
+.container-bluetooth input[type="radio"] + label::before {
+    content: url(.//img/radio-off.svg);
+    display: inline-block;
+    position: absolute;
+    left: 0;
+    top: -2px; 
+}
+.container-bluetooth input[type="radio"]:checked + label::before {
+    content: url(.//img/radio-on.svg);
+    display: inline-block;
+    position: absolute;
+    left: 0;
+    top: -2px; 
+}
+.container-bluetooth input[type="radio"]:hover + label::before {
+    opacity: 0.6;
+}
+.container-bluetooth input[type="radio"]:focus + label::before {
+    opacity: 1;
+}
+.filter-button {
+    width: 200px;
```

Comment:
maybe max-width?

Reply:
max-width squeeze the button to it's content. Maybe I should use width=100%?

Reply:
used width - 100%